src/collections/_documentation/performance-monitoring/configuration/node.md

```diff
@@ -94,6 +94,54 @@ Spans are instrumented for the following operations within a transaction:
 - `get` calls using native `http` and `https` modules
 - Middleware (Express.js only)
 
+**Trace context for Errors**
+
+By default, Sentry error events will not get trace context unless you configure the scope with the transaction. For example:
+
+```js
+const Sentry = require("@sentry/node");
+const Apm = require("@sentry/apm");
+
+const http = require('http');
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracesSampleRate: 1.00,
+  integrations: [
+    // enable HTTP calls tracing
+    new Sentry.Integrations.Http({ tracing: true }),
+  ],
+});
+
+const transaction = Sentry.startTransaction({
+  op: "transaction",
+  name: 'My Transaction'
+});
+
+Sentry.configureScope(scope => {
+  scope.setSpan(transaction);
+});
```

Comment:
```suggestion
// Note that we set the Transaction as the Span on the scope
// This step makes sure that in case an error happens during the lifetime of the transaction
// The Transaction context will be attached to the error event
Sentry.configureScope(scope => {
  scope.setSpan(transaction);
});
```